ovos_tts_server/gradio_app.py

```diff
@@ -54,7 +53,7 @@ def bind_gradio_service(app, tts_engine, title, description, info, badge,
             [textbox, radio],
             [audio],
         )
-        radio.change(lambda lang: CoquiTTS.langs[lang]["sentence"],
-                     radio, textbox)
+        # radio.change(lambda lang: CoquiTTS.langs[lang]["sentence"],
```

Comment:
this can later be some pre defined list in this repo, a .txt per language maybe with some examples